go.work

```diff
@@ -5,6 +5,7 @@ use (
 	./api
 	./client/v2
 	./core
+	./cosmovisor
```

Comment:
@faddat, meaculpa, removing this was breaking `make test`. We have a few solutions to fix that:

- In `run-tests` (Makefile), as `GOWORK=off` before running `go test`. This requires (although I think it's good), packages to be properly tagged to pass CI.
- Accept that it will upgrade the tendermint version until the SDK is downgrading to 34.x on main as well (as the support of 0.35 is anyway dropped by TM)

Reply:
I am actually for doing both, merging with cosmovisor in the go.work but then still disable it in the CI runs to force us to tag properly our versions.